### PR TITLE
Add test instructions

### PR DESCRIPTION
**Pull request by javiermtorres**
## What's changing

Add more detailed instructions for testing the SDK and the backend.

## How to test it

Check that the instructions match our current practice and the CI.

## Additional notes for reviewers

N/A

## I already...

- [x] added some tests for any new functionality
  - No auto tests needed 
- [x] updated the documentation
- [x] checked if a (backend) DB migration step was required and included it if required
  - Doc changes only, no DB changes

### CODE REVIEW

**Comment by javiermtorres** [comment]
> Do you think (in other places too) we should make it clear that when the user does `SQLALCHEMY_DATABASE_URL=sqlite:///local.db ...` to run tests, it creates (or uses if it already exists) a SQLite DB that isn't removed afterwards?

It's a good point :-/ Maybe in a separate PR we should add a step in tests to remove this file, plus set this env var directly as a fixture (and maybe use `sqlite:////tmp/local.db` instead to minimize possible reuse of the file).

So yes, I'll add a line about it, at the very least.

File: lumigator/python/mzai/sdk/README.md [hunk 1]
```diff
@@ -15,8 +15,26 @@ or
...
+```
+
+To run integration tests, please use:
+
+```bash
+pushd ../../../../ # go back to the project root
+make start-lumigator-build # wait until all containers are up and running
+popd
+uv run pytest -o python_files="int_test_*.py"
+```
```
[the remark this answers is not in the document]
Comment (javiermtorres) [reply lumigator/python/mzai/sdk/README.md:40]: Hmmmmm..... Yeah, it's clearer. I'll add that too.

**Review by davidmanzanoai — APPROVED** [review]
I agree with the changes.

**Comment by javiermtorres** [comment]
@peteski22 @davidmanzanoai @dpoulopoulos if it's ok with you, in another PR:
* I'll move the dev.md guide to somewhere else (I'd propose `get-started`). I'd also like to do some light edits to clarify some issues, mainly the DB handling
* I'll reference the `README`s from `dev.md` to avoid doc duplication, when talking about testing the backend and the SDK. @dpoulopoulos should we copy the `README`s content for doc building?

If you agree with the current status plus the scope of the proposed future PR, please approve this PR. Thanks in advance,

**Pull request merged**